tests/components/wiz_light/test_config_flow.py

```diff
@@ -0,0 +1,159 @@
+"""Test the WiZ Light config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries, setup
+from homeassistant.components.wiz_light.config_flow import (
+    WizLightConnectionError,
+    WizLightTimeOutError,
+)
+from homeassistant.components.wiz_light.const import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_NAME
+
+from tests.common import MockConfigEntry
+
+FAKE_BULB_CONFIG = '{"method":"getSystemConfig","env":"pro","result":\
+    {"mac":"ABCABCABCABC",\
+    "homeId":653906,\
+    "roomId":989983,\
+    "moduleName":"ESP_0711_STR",\
+    "fwVersion":"1.21.0",\
+    "groupId":0,"drvConf":[20,2],\
+    "ewf":[255,0,255,255,0,0,0],\
+    "ewfHex":"ff00ffff000000",\
+    "ping":0}}'
+
+TEST_SYSTEM_INFO = {"id": "ABCABCABCABC", "name": "Test Bulb"}
+
+
+TEST_CONNECTION = {CONF_HOST: "1.1.1.1", CONF_NAME: "Test Bulb"}
+
+
+async def test_form(hass):
+    """Test we get the form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
```

Comment:
If the source is changed to SOURCE_IMPORT, it would test both async_step_user and async_step_import, picking up another line of coverage for the test.